### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by ehteshamkafeel**
## Description
Release 1.1.0 for `parseState` and getters changes.

## Reviewers
- [x] @andy-armstrong 

If you've been tagged for review, please check your corresponding box once you've given the :+1:.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.1.0 (2016-05-26)
+* Removed parseState method in favor of PageableCollection
+* Added getters to get the state attributes
```
Comment (dan-f) [inline CHANGELOG.md]: Could you clarify that these two notes relate to the `PagingCollection` module?

**Comment by dan-f** [comment]
👍  from me other than my one comment. Thanks!!

**Comment by dan-f** [comment]
Oops, once again, I'm responding to the mention bot.

**Comment by andy-armstrong** [comment]
👍

**Comment by ehteshamkafeel** [comment]
@dan-f Thanks, no problem this time the PR was ready. Good to have your comments too.

**Pull request merged**